content/pages/privacy-policy.md

```diff
@@ -0,0 +1,105 @@
+
+Crash reports are handled by the Stores/Vendors of each platforms.
+Please refer to those platforms for information about the privacy policies.
+Depending on the platform, Mixxx Core Developers have access to these reports, which usually contain:
+
+- Operating System version
+- CPU and GPU
+- Symbolic StackTrace of the process
+- List of DLLs loaded by the process
+
+None of these data is stored on Mixxx servers.
+
+## Mixxx update mechanism
+
+The Mixxx application itself doesn't provide an update mechanism.
+
+Updates are handled by the Stores of each platforms, where the user has downloaded Mixxx from.
+Please refer to those platforms for information about the privacy policies.
+
+## Mixxx metadata requests
+
+Mixxx can automatically search online for metadata associated to your media, to 3rd parties.
+Those requests can contain information about the media, including artist, title, album, etc...
+This metadata mechanism can be disabled at every moment, in Mixxx preferences.
+
+The main 3rd parties that can be contacted by Mixxx, automatically is:
+
+ - Mixxx MusicBrainz instance.
+
+Please refer to those services for their own privacy policy.
+Mixxx does not have access to any of those requests and data, except in the MusicBrainz case, detailed below, in this document.
+Mixxx mobile
+
+## Main Mixxx website
+
+The main website of Mixxx, aka www.mixxx.org, does not create or use any user account.
+There are no registration forms, or any similar mechanism.
+Therefore, no personal information is stored at any moment, about the user, by Mixxx.
+
+## Community Mixxx websites
+
+The community websites from Mixxx are related to development and community, and are the following:
+
+- GitHub
+- Discourse Forum
+- Zulip Chat
+
+Each of those websites have different users system, but work in a similar way.
+
+No user information is processed in any way other than being able to log in and have a profile.
+
+On GitHub no user information is shared to any 3rd party.
+
+Our Discourse Forum has an own Privacy Policy. You can find it at: [https://mixxx.discourse.group/privacy](https://mixxx.discourse.group/privacy)
+
+One can modify their user information at any time, through the options of those websites.
+
+One can request deletion of their account, by contacting Mixxx.
+
+## MusicBrainz
+
+Mixxx is running a MusicBrainz instance, that can be used to find information about a music, and artist and so on.
+
+Those automatic requests can contain information about the media, including artist, title, album, etc...
+
+This metadata mechanism is accepted on the first run of Mixxx, and can be disabled at every moment, in Mixxx preferences.
+
+The information sent does not contain any name, username, email from the user.
+
+No information from the request is kept after the transaction.
+
+The connection logs can contain some limited information (like user-agents) and partial search next to the IP.
+
+Last update date: 27 June 2023.

```

Comment:
Can this go into the page metadata? If it should also be displayed here, let's make it italic:


```suggestion
*Last updated: 27 June 2023*
```